tests/handlers/aws/test_logstash_output.py

```diff
@@ -128,3 +137,32 @@ def test_sent_messages(self) -> None:
 
         msgs = self.logstash.get_messages()
         assert len(msgs) == 2
+
+    def test_failure_sending_messages(self) -> None:
+        os.environ["S3_CONFIG_FILE"] = _prepare_config_file(
+            self,
+            "config.yaml",
+            dict(CloudwatchLogStreamARN=self.cloudwatch_group_arn, LogstashURL="http://fake.url"),
+            "folder/config2.yaml",
+        )
+
+        event_cloudwatch_logs, event_ids_cloudwatch_logs = _logs_retrieve_event_from_cloudwatch_logs(
+            self.logs_client, group_name=self.group_name, stream_name=self.stream_name
+        )
+
+        ctx = ContextMock(TIMEOUT_15m)
+        handler(event_cloudwatch_logs, ctx)  # type: ignore
```

Comment:
please. let's keep the assertion on the expected output of the handler

in general I see that in the logstash integration tests we are reducing the number of assertions

Reply:
I added assertion to check message content, not just number of received messages.

Reply:
I would add full assertions on every field of the payload: `aws`, `cloud`, `log` and `message`